frontend/src/contexts/AuthContext.tsx

```
@@ -0,0 +1,146 @@
+      return response.data;
+    } catch (error) {
+      throw error;
+    }
+  };
+
+  const enable2FA = async () => {
+    try {
+      const response = await api.post('/auth/2fa/enable');
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)